src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -924,9 +973,10 @@ __meta__ = {
 }
 
 [node name="CellEditor" type="Control" parent="MicrobeEditorGUI"]
-visible = false
```

Comment:
Not really important as it's overriden ingame but for consistency the initial shown tab should be the report tab in the godot editor.

Reply:
This was reset to display the Report tab by default